docs/developer/release/README-release-process.html

```diff
@@ -239,7 +240,42 @@ <h2 id="continuous-integration">Continuous integration tests</h2>
     <li>Daikon: <a href="https://dev.azure.com/mernstdaikon/daikon/_build/latest?definitionId=1&branchName=master"><img src="https://dev.azure.com/mernstdaikon/daikon/_apis/build/status%2Fcodespecs.daikon?branchName=master" alt="codespecs/daikon Azure Pipelines status"></a></li>
   </ul>
 
-  <h2 id="snapshot_release">Snapshot release</h2>
+  <h2 id="snapshot_release">Snapshot release to local Maven repository</h2>
+  <p>This section explains how to release the Maven artifacts to a local Maven repository.</p>
```

Comment:
I would not call this a release, it's just deploying to the local Maven repository.  We document how to do this for Maven in [the manual](https://checkerframework.org/manual/#maven-locally-built).  I think the Gradle section below might be better to put in [the Gradle plugin in documentation](https://github.com/kelloggm/checkerframework-gradle-plugin#specifying-a-checker-framework-version).  That section already shows how to use a locally built version.